digdag-standards/src/main/java/io/digdag/standards/command/EcsCommandExecutor.java

```diff
@@ -307,6 +308,13 @@ public void cleanup(
         }
     }
 
+    private long maxWaitForFetchLogEventsInSec = 60L;
+
+    @VisibleForTesting
+    void setMaxWaitForFetchLogEvents(long sec) {
+        this.maxWaitForFetchLogEventsInSec = sec;
```

Comment:
Why don't you get this value from system config with a key like `CONFIG_RETRY_TASK_SCRIPTS_DOWNLOADS` so the test code can pass a shorter value via system config?

Reply:
fixed https://github.com/treasure-data/digdag/pull/1603/commits/9bf8a447d7bb054e220cf7031a0da700074cadf7